app/views/layouts/rails_admin/_head.html.erb

```diff
@@ -1,7 +1,7 @@
-<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
-<meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
-<meta content="width=device-width, initial-scale=1" name="viewport; charset=utf-8"/>
-<meta content="NONE,NOARCHIVE" name="robots"/>
+<meta content="IE=edge" http-equiv="X-UA-Compatible">
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+<meta content="width=device-width, initial-scale=1" name="viewport; charset=utf-8">
+<meta content="NONE,NOARCHIVE" name="robots">
```

Comment:
In the examples at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#examples it uses the self-closing tags though:

![image](https://github.com/railsadminteam/rails_admin/assets/1435346/4d318a80-6388-4139-b5c9-47b44a46e266)